src/applications/personalization/dashboard/components/ManageYourVAHealthCare.jsx

```diff
@@ -157,17 +157,19 @@ const ManageYourVAHealthCare = ({
       <PrescriptionsWidget />
     </DowntimeNotification>
     {isEnrolledInHealthCare &&
-      !showCernerWidget && <ScheduleAnAppointmentWidget />}
+      !showCernerAppointmentWidget && <ScheduleAnAppointmentWidget />}
     {isEnrolledInHealthCare &&
-      showCernerWidget && <ScheduleAnAppointmentCernerWidget />}
+      showCernerAppointmentWidget && <ScheduleAnAppointmentCernerWidget />}
   </>
 );
 
 const mapStateToProps = state => {
   const isEnrolledInHealthCare = isEnrolledInVAHealthCare(state);
   const hcaEnrollmentStatus = selectEnrollmentStatus(state);
   const showServerError = hasESRServerError(state);
-  const showCernerWidget = selectIsCernerPatient(state);
+  const showCernerAppointmentWidget = !!selectCernerAppointmentsFacilities(
```

Comment:
renaming this since we will soon also have `showCernerRxWidget` and `showMessagingAppointmentWidget `